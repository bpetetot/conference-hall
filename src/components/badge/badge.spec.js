/* eslint-env jest */
import snap from 'tests/snapshot'
import Badge from './index.js'

const snapshot = (props) => snap(Badge)({ ...props })

describe('components/badge', () => {
  it('should not render when no children', snapshot({}))
  it('should render with children', snapshot({ children: 'Benjamin', className: 'test' }))
})
