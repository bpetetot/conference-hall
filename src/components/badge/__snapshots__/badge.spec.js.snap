// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/badge should not render when no children 1`] = `null`;

exports[`components/badge should render with children 1`] = `
<span
  className="cc-badge cc-badge-primary test"
  style={undefined}
>
  Benjamin
</span>
`;
